api/spec/src/cloud/subjects.tsp

```diff
@@ -80,5 +80,5 @@ model SubjectUpsert {
   currentPeriodEnd?: DateTime;
 
   @example("cus_JMOlctsKV8")
-  stripeCustomerId?: string;
+  stripeCustomerId?: string | null;
```

Comment:
Probably should be changed in the Subject model as well?

Reply:
I do not think it would be really necessary, but I'll add it there as well in order to be on the safe side.